src/Vector2_TEST.py

```diff
@@ -0,0 +1,262 @@
+# 
+  # Copyright (C) 2021 Open Source Robotics Foundation
+  #  
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #  
+  #       http://www.apache.org/licenses/LICENSE-2.0
+  #  
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+  #
+#
+
+import unittest
+import math
+from ignition.math import Vector2d
+
+class Vector2Test(unittest.TestCase):
+
+  def test_construction(self):
+    v = Vector2d()
+    self.assertAlmostEqual(0.0, v.X())
+    self.assertAlmostEqual(0.0, v.Y())
+
+    vec = Vector2d(1, 0)
+    self.assertEqual(vec.X(), 1)
+    self.assertEqual(vec.Y(), 0)
+
+    # Copy constructor
+    vec2 = Vector2d(vec)
+    self.assertEqual(vec2, vec)
+
+    # Copy operator
+    vec3 = vec
+    self.assertEqual(vec3, vec)
+
+    # Inequality
+    vec4 = Vector2d()
+    self.assertNotEqual(vec, vec4)
+
+  def test_vector2(self):
+    # Constructor
+    v = Vector2d(1, 2)
+    self.assertAlmostEqual(1.0, v.X())
+    self.assertAlmostEqual(2.0, v.Y())
+
+    # Distance
+    self.assertAlmostEqual(2.236, v.Distance(Vector2d()), delta=1e-2)
+
+    # Normalize
+    v.Normalize()
+    self.assertTrue(v.Equal(Vector2d(0.447214, 0.894427), 1e-4))
+
+    # Set
+    v.Set(4, 5)
+    self.assertTrue(v.Equal(Vector2d(4, 5), 1e-4))
+
+    # Operator GetAbs
+    v.Set(-1, -2)
+    self.assertTrue(v.Abs().Equal(Vector2d(1, 2), 1e-4))
+
+    # _eq_
```

Comment:
Given that all the VectorX_TEST.py are analogous try to use the same nomenclature when describing the analogous methods.

Reply:
Addressed